src/types/emr/specimenDefinition/specimenDefinition.ts

```diff
@@ -71,11 +71,10 @@ export const RETENTION_TIME_UNITS = [
 
 export interface TypeTestedSpec {
   is_derived: boolean;
-  specimen_type: Code | null;
   preference: Preference;
   container: ContainerSpec | null;
   requirement: string | null;
-  retention_time: DurationSpec | null;
+  retention_time?: DurationSpec | null;
   single_use: boolean | null;
```

Comment:
```suggestion
  single_use?: boolean | null;
```

<img width="515" alt="image" src="https://github.com/user-attachments/assets/fc8799c2-38c3-464b-95e4-b5a060a7594d" />